plugins/kubernetes-backend/src/types/types.ts

```diff
@@ -118,8 +118,13 @@ export interface KubernetesFetcher {
 }
 
 // Used to locate which cluster(s) a service is running on
-export interface KubernetesClusterLocator {
-  getClusterByServiceId(serviceId: string): Promise<ClusterDetails[]>;
+export interface KubernetesServiceLocator {
+  getClustersByServiceId(serviceId: string): Promise<ClusterDetails[]>;
```

Comment:
Still not super thrilled about the `serviceId`. Thoughts on simply passing an `Entity` here instead?

Reply:
I think this makes sense, we are doing a POST now we should post the entity with the auth details.

Reply:
Yep, although we can also have the layer outside this go fetch the entity data from the catalog by name too.

```ts
router.get('/mah-resources/:entityName', (req, res) => {
  const entity = await catalogClient.getEntityByName(req.params.entityName);
  const clusters = await serviceLocator.getClustersByEntity(entity);
  ...
})
```